codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/GenerationContext.java

```diff
@@ -56,38 +60,48 @@ public GenerationContext(RubySettings rubySettings,
         this.protocol = protocol;
         this.protocolGenerator = protocolGenerator;
         this.applicationTransport = applicationTransport;
+        this.symbolProvider = symbolProvider;
     }
 
-    public RubySettings getRubySettings() {
+    @Override
+    public Model model() {
+        return model;
+    }
+
+    @Override
+    public RubySettings settings() {
         return rubySettings;
     }
 
-    public FileManifest getFileManifest() {
-        return fileManifest;
+    @Override
+    public SymbolProvider symbolProvider() {
+        return symbolProvider;
     }
 
-    public List<RubyIntegration> getIntegrations() {
-        return integrations;
+    @Override
+    public FileManifest fileManifest() {
+        return fileManifest;
     }
 
-    public Model getModel() {
-        return model;
+    public List<RubyIntegration> integrations() {
+        return integrations;
     }
 
-    public ServiceShape getService() {
+    public ServiceShape service() {
         return service;
     }
 
-    public ApplicationTransport getApplicationTransport() {
+    public ApplicationTransport applicationTransport() {
```

Comment:
These don't follow Java convention of getThing. But I assume you did this because to be consistent with CodeGen context?